pam/integration-tests/cli-integration_test.go

```diff
@@ -18,9 +18,30 @@ import (
 
 var daemonPath string
 
+func prepareCLITest(t *testing.T) {
+	t.Helper()
+
+	// Due to external dependencies such as `vhs`, we can't run the tests in some environments (like LP builders), as we
+	// can't install the dependencies there. So we need to be able to skip these tests on-demand.
+	if os.Getenv("AUTHD_SKIP_EXTERNAL_DEPENDENT_TESTS") != "" {
+		t.Skip("Skipping tests with external dependencies as requested")
+	}
+
+	// This happens when GO_WANT_HELPER_PROCESS is defined, in such case let's just skip.
+	if daemonPath == "" {
+		t.Skip("Test cannot run without an example daemon")
+	}
```

Comment:
This can be removed. When we define `GO_WANT_HELPER_PROCESS`, it means we are running a specific test to mock something (in this case, we run `TestMockGPasswd` to mock the gpasswd command). This env is just to control the behaviour of that specific test (as in "if we didn't set this, this shouldn't be executed as it is not a real test") and of TestMain (since TestMain is always executed before any test).

TLDR: We only care about `GO_WANT_HELPER_PROCESS` in TestMain and in TestMockGPasswd.